numpy/core/src/multiarray/arraytypes.c.src

```diff
@@ -3124,7 +3127,32 @@ finish:
  **                                 ARGFUNC                                 **
  *****************************************************************************
  */
+#if defined( __arm__ ) || defined(__aarch64__)
```

Comment:
this going to break the build on `armhf`, you need to specfiy compiler flag `-mcpu=neon` at least also `neon` isn't available in all arm7 chips, so it must be dispatched in runtime by default

Reply:
changed to NPY_CPU_HAVE

Reply:
`NPY_CPU_HAVE` can't be used within a preprocessor, please remove it and use following guard
```C
#if defined(__ARM_NEON__) || defined (__ARM_NEON)
````
this will be always activated on aarch64 but when it comes to armhf the user has to pass flags `-mcpu=neon` within `CFLAGS`.

Reply:
@seiko2plus Did you actually means following guard?
`#if defined(NPY_HAVE_NEON) || defined(__ARM_NEON__)`

Reply:
no, `NPY_HAVE_NEON` defined by #13516. just replace
```C
    #ifdef NPY_CPU_HAVE({NEON})
    #ifdef __ARM_NEON__
```
with
```C
#if defined(__ARM_NEON__) || defined (__ARM_NEON)
````

Reply:
yes, It worked.